### PR TITLE
[NG-16] - TEST

### PR DESCRIPTION
**Pull request by VitoAlbano**
**Please check if the PR fulfills these requirements**

> - [ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/src/lib/form/components/widgets/core/container.model.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ import { FormFieldTypes } from './form-field-types';
 
 export class ContainerModel extends FormWidgetModel {
 
-    field: FormFieldModel;
+    declare field: FormFieldModel;
```
Comment (DenysVuika) [inline lib/core/src/lib/form/components/widgets/core/container.model.ts:27]: migration error I think

File: lib/core/src/lib/form/components/widgets/core/form-field.model.ts [hunk 2]
```diff
@@ -42,12 +42,12 @@ export class FormFieldModel extends FormWidgetModel {
     readonly defaultDateTimeFormat: string = 'D-M-YYYY hh:mm A';
 
     // model members
-    fieldType: string;
-    id: string;
-    name: string;
-    type: string;
+    declare id: string;
```
Comment (DenysVuika) [inline lib/core/src/lib/form/components/widgets/core/form-field.model.ts:45]: the migrator had a bad day

**Comment by DenysVuika** [comment]
@VitoAlbano we can incorporate some of the changes earlier, to simplify the migration

**Comment by VitoAlbano** [comment]
@DenysVuika This was just a test in the end after a little bit of experimentation i decided to go step by step, i've created a new PR to move ADF to NG15 and i'll do the same shortly on Apps

**Comment by VitoAlbano** [comment]
Closing this as the changes are too disruptive and some of them needs to be checked

**Pull request closed without merging**